src/js/CloudExplorer.jsx

```diff
@@ -162,6 +162,15 @@ export default class CloudExplorer extends React.Component {
             files,
             loading: false
           });
+          // the first display, single service mode, try to enter
+          // this is useful when CE is used by hosting companies to display the user files, and the user has logged in their system
+          if(!this.initDone && files.length === 1 && path.length === 0/* && files[0].isLoggedIn === true*/) {
```

Comment:
I think initDone should be part of the component state.

Reply:
but more generally, I don't quite understand this part.

Reply:
yes again that's a way to open a service when CE starts, so only the 1st time and if it is "/" which is `ls`
maybe i should put all this somewhere else...